website/docs/interactors/3-locators-filters-actions.md

```diff
@@ -52,68 +50,81 @@ TextField('Username:', { id: 'username-id' }).exists();
  _See the source code of the TextField interactor [here](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/text-field.ts)_.
 :::
 
-You can think of locators as the "default filter" because filters and locators both serve the same functionality. The reason why we offer both solutions is for your convenience because having to pass in an object for each interactor can be repetitive.
+You can think of the locator as the "default filter" because filters and locators both provide the same functionality. The reason why BigTest offers both solutions is convenience, because having to pass in an object for each interactor can get repetitive – unless you use a filter.
```

Comment:
> because having to pass in an object for each interactor can get repetitive
This was to point out that `Button('Submit')` is more convenient than having to use filters like `Button({ value: 'Submit' })` each time.

So adding `- unless you use a filter` causes confusion.